internal/controllers/machineset/machineset_controller.go

```diff
@@ -427,7 +427,7 @@ func (r *Reconciler) reconcileDelete(ctx context.Context, s *scope) (ctrl.Result
 	// else delete owned machines.
 	for _, machine := range machineList {
 		if machine.DeletionTimestamp.IsZero() {
-			log.Info("Deleting Machine", "Machine", klog.KObj(machine))
+			log.Info("Deleting Machine (MS deleted)", "Machine", klog.KObj(machine))
```

Comment:
```suggestion
			log.Info("Deleting Machine (MachineSet deleted)", "Machine", klog.KObj(machine))
```

What do you think about spelling it out? Users may not immediately recognize the abbreviation.

Reply:
Sure, I wiill do

Reply:
This was done, resolving